src/Teams/Data/Repositories/QueuedProgramRepository.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Teams.Domain;
+using Teams.Models;
+
+namespace Teams.Data.Repositories
+{
+    public class QueuedProgramRepository : IQueuedProgramRepository
+    {
+        private ApplicationDbContext _db;
+        public QueuedProgramRepository(ApplicationDbContext db)
+        {
+            _db = db;
+        }
+        public void Add(ProgramCodeQuestion question, string text)
```

Comment:
Let's pass only what we need. We need only questionId not the whole ProgramCodeQuestion.